idl/response/foxsi_get_spectral_image.pro

```diff
@@ -0,0 +1,378 @@
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;;FUNCTION:      "foxsi_get_spectral_image"
```

Comment:
This function name is incredibly confusing. Would recommend something like foxsi_get_default_spectral_image.pro or something.